LEAF_Request_Portal/utils/db_upgrade/Update_RMC_DB_2020011000-20200207.sql

```diff
@@ -0,0 +1,13 @@
liance with required security requirements.</li>\r\n                <li>I will protect sensitive information from unauthorized disclosure, use, modification, or destruction, including using encryption products approved and provided by the VA to protect sensitive data.</li>\r\n                <li>I will not store or transport any VA sensitive information on any portable storage media or device unless it is encrypted using VA approved encryption.</li>\r\n                <li>I will use VA-provided encryption to encrypt any e-mail, including attachments to the e-mail, that contains VA sensitive information before sending the e-mail.  I will not send any e-mail that contains VA sensitive information in an unencrypted form.  VA sensitive information includes personally identifiable information and protected health information.</li>\r\n                <li>I may be required to acknowledge or sign additional specific or unique rules of behavior in order to access or use specific VA systems.  I understand that those specific rules of behavior may include, but are not limited to, restrictions or prohibitions on limited personal use, special requirements for access or use of the data in that system, special requirements for the devices used to access that specific system, or special restrictions on interconnections between that system and other IT resources or systems.</li>\r\n            </ol>\r\n        </li>\r\n        <li>Acknowledgement and Acceptance\r\n            <ol type=\"a\">\r\n                <li>I acknowledge that I have received a copy of these Rules of Behavior.</li>\r\n                <li>I understand, accept and agree to comply with all terms and conditions of these Rules of Behavior.</li>\r\n            </ol>\r\n        </li>\r\n    </ol>\r\n</div>\r\n<div>\r\n<input class=\"icheck_rob\" type=\"checkbox\" id=\"rob_acceptance\" value=\"Accepted terms and rules of behavior\" style=\"width: 32px\" /> I understand and accept. <span id=\"rob_required\" style=\"margin-left: 8px; color: red\">*&nbsp; Required</span>\r\n</div>', `htmlPrint` = '<style>\r\n    #devconsole_description > p, #devconsole_description > ul > li {\r\n        font-size: 16px;\r\n    }\r\n</style>\r\n<div id=\"devconsole_description\">\r\n<p><b>This is a request to access the LEAF Developer Console.</b></p>\r\n<p>Approved individuals will gain the ability to modify LEAF\'s user interface using HTML, CSS, and JavaScript technologies.</p>\r\n<p>By approving this request:\r\n<ul>\r\n    <li>I validate that the person requesting access as the technical ability to work with HTML, CSS, and JavaScript source code.</li>\r\n    <li>I recognize that source code customizations and their maintenance are the responsibility of the office seeking to make customizations. In the event that the responsible office becomes unable to support maintenance, customizations may be easily removed to restore standard functionality.</li>\r\n</ul>\r\n</p>\r\n</div>' WHERE `indicators`.`indicatorID` = -5;
```

Comment:
Fortify is flagging this line.

Reply:
This file should be renamed to match the convention unless conventions are changing again.